test/e2e-tests.sh

```diff
@@ -28,27 +28,32 @@ source $(dirname $0)/../scripts/e2e-tests.sh
 # Read metadata.json and get value for key
 # Parameters: $1 - Key for metadata
 function get_meta_value() {
-  go run ${REPO_ROOT_DIR}/vendor/knative.dev/pkg/testutils/metahelper --get $1
+  go run "${REPO_ROOT_DIR}/vendor/knative.dev/pkg/testutils/metahelper" --get "$1"
 }
 
 function knative_setup() {
   start_latest_knative_serving
 }
 
-# Run a prow-cluster-operation tool
+# Run "kntest cluster" tool
 # Parameters: $1..$n - parameters passed to the tool
 function run_prow_cluster_tool() {
-  go run ${REPO_ROOT_DIR}/vendor/knative.dev/pkg/testutils/clustermanager/prow-cluster-operation $@
+  if [[ "${REPO_NAME}" == "test-infra" ]]; then
```

Comment:
Isn't this file only run in test-infra?
Also, this is exactly the kind of BS that kntest is supposed to get rid of. I think we should just get it into prow-tests immediately.

Reply:
`kntest` is still under active development, so at the staging I would prefer to install it as part of the CI flow to always get its latest version, otherwise we would need to update the `prow-tests` image every time we have update `kntest`. It would be super annoying as now we don't have the automated process to update the images.

For why we have that `if` logic, in this repo we always run the local version of `kntest` but for other repos they need to `go get`, if we also use `go get` in this repo, we won't be able to run and test `kntest` changes in this PR.

Reply:
I think one of us has this swapped -- this is test/e2e-tests.sh not scripts/e2e-tests.sh

This file is only used in test-infra AFAICT?

Reply:
Correct, this file is now only used in test-infra and is run to make sure it works in presubmit tests.
I will have followup PRs to move the changes to `script/e2e-tests.sh` so that all other Knative projects can get this update.